understanding/22/accessible-authentication-minimum.html

```diff
@@ -141,70 +140,67 @@ <h3>Hiding characters</h3>
 
     </section>
     <section id="benefits">
-        <h2>Benefits of Accessible Authentication</h2>
-
-        <p>People with cognitive issues relating to memory, reading (for example, dyslexia), numbers (for example, dyscalculia), or perception-processing limitations will be able to authenticate irrespective of the level of their cognitive abilities.</p>
-    </section>
-
-    <section id="examples">
-        <h2>Examples of Accessible Authentication (Minimum)</h2>
-        <p>The examples of this Success Criterion are the same as the <a href="accessible-authentication-enhanced.html#examples">Accessible Authentication (Enhanced)</a> examples.</p>
-
+      <h2>Benefits of Accessible Authentication (Minimum)</h2>
+      
+      <p>People with cognitive issues relating to memory, reading (for example, dyslexia), numbers (for example, dyscalculia), or perception-processing limitations will be able to authenticate irrespective of the level of their cognitive abilities.</p>
+      
+   </section>
+   
+   <section id="examples">
+      <h2>Examples of Accessible Authentication (Minimum)</h2>
+      <p>The examples of this Success Criterion are the same as the <a href="accessible-authentication-enhanced.html#examples">Accessible Authentication (Enhanced)</a> examples.</p>
+
+      <ul>
+         <li>A web site uses a properly marked up username (or email) and password fields as the login authentication (meeting <a href="identify-input-purpose">Success Criterion 1.3.5 Input Purpose</a> and <a href="name-role-value">Success Criterion 4.1.2: Name, Role, Value</a>). The user's browser or integrated third-party password manager extension can identify the purpose of the inputs and automatically fill in the username and password.</li>
+         <li>A web site does not block paste functionality. The user is able to use a third-party password manager to store credentials, copy them, and paste them directly into a login form.</li>
+         <li>A web site uses WebAuthn so the user can authenticate with their device instead of username/password. The user's device could use any available modality. Common methods on laptops and phones are facial-scan, fingerprint, and PIN (Personal Identification Number). The web site is not enforcing any particular use, it is assumed a user will setup a method that suits them.</li>
```

Comment:
```suggestion
         <li>A web site uses WebAuthn so the user can authenticate with their device instead of username/password. The user's device could use any available modality. Common methods on laptops and phones are facial-scan, fingerprint, and PIN (Personal Identification Number). The web site is not enforcing any particular use; it is assumed a user will setup a method that suits them.</li>
```